data/virtual/kvm.yaml

```diff
@@ -0,0 +1,4 @@
+---
+svckill::ignore_defaults:
+  - ^ovirt-.*
```

Comment:
yes, ovirt-guest-agent and qemu-guest-agent are on my ovirt vms used for testing, and yes that's the point. 
the qemu-guest-agent is only on libvirt vms, while ovirt-guest-agent is also on ovirt vms; but there is no fact to differentiate the two.